interview-questions.md

```diff
@@ -6,44 +6,47 @@ First, without external resources. Challenge yourself to answer from memory.
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely on your own, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
-1. What is object-oriented programming? How is it different than functional programming?
+1. What is object-oriented programming? 
 
-Your answer:
+Your answer: How is it different than functional programming? OOP is a computer programming model that organizes data and objects to make it easier to maintain. The output to Functional programming is the same as the inputs
 
-Researched answer:
+Researched answer: OOP is a paradigm which uses objects to represent things you are programing, for example data structures, the objects hold data such as keys and values, and you can add more data to it while on the other hand FP works best with there is nothing else needed to be added to the function, for example getSum() and with preset input data.
 
 2. What is the difference between a Float and an Integer in Ruby?
 
-Your answer:
+Your answer: An integer would be a whole number, and a float would be one with a decimal, you can use an integer if you are dealing with whole numbers, but if you want to see tax, you would use a float. So use either or for your desired needs/ desired outcomes
 
-Researched answer:
+Researched answer: Google basically said what I said so I think I hit this one right on the nail!
 
-3. Ruby has an implicit return. What does that mean?
+3. Ruby has an implicit return. What does that mean? 
 
-Your answer:
+Your answer: A function that is returning values automatically without having the word "return" after the function.
 
-Researched answer:
+Researched answer: Return is implied, mostly used when the function body has a single expression.
```

Comment:
Ruby will automatically use implicit return which will return the last line of any method unless otherwise stated.